setup.py

```diff
@@ -1,23 +1,24 @@
+#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-    Setup file for core.
+    Setup file for f0cal.
     Use setup.cfg to configure your project.
 
-    This file was generated with PyScaffold 3.2.3.
+    This file was generated with PyScaffold 3.1.
     PyScaffold helps you to put up the scaffold of your new Python project.
     Learn more under: https://pyscaffold.org/
 """
 import sys
 
-from pkg_resources import VersionConflict, require
+from pkg_resources import require, VersionConflict
 from setuptools import setup
 
 try:
-    require('setuptools>=38.3')
+    require("setuptools>=38.3")
 except VersionConflict:
     print("Error: version of setuptools is too old (<38.3)!")
     sys.exit(1)
 
 
 if __name__ == "__main__":
-    setup(use_pyscaffold=True)
+    setup(use_pyscaffold=False)
```

Comment:
Do you know why this line is switched?

Reply:
Don't think that was me: https://github.com/f0cal/core2/commit/1268e4bb151e9802a81fed19c749eb710dbdb6b0#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7